### ISSUE TITLE
Accepted parameter of Unsplash class doesn't match the index.d.ts declaration

### ISSUE THREAD

**Issue by BenjiTheC** [unsplash/unsplash-js#121]
#### Steps to Reproduce

1. Instantiate the unsplash instance with `applicationId` as option key (per documented in the `index.d.tx`

```js
const Unsplash = require('unsplash-js').default;
global.fetch = require('node-fetch');

const unsplash = new Unsplash({
    applicationId: ACCESS_KEY,
    secret: SECRET_KEY,
});
```

Try fetch the photos

```js
unsplash.photos.listPhotos()
  .then(res => res.json())
  .then(json => console.log(json))
  .catch(e => console.log(e));
```

#### Observed Behaviour

Getting an error message `{ errors: [ 'OAuth error: The access token is invalid' ] }` 

But after I change the key that accepts the access token from `applicationId` to `accessKey` as described in the current README, I'm able to get the desired result.

I believe this should be an error brought in recently - `applicationId` worked fined for me less than a month ago.

**Comment by lukechesser** [comment]
Hm, we removed the [use of `applicationId` in 6.0](https://github.com/unsplash/unsplash-js/blob/master/CHANGELOG.md#changes) to avoid confusion. Where are you seeing `applicationId` still listed?

**Comment by samijaber** [comment]
closed in #143
